near-contract-standards/src/fungible_token/mod.rs

```diff
@@ -1,3 +1,5 @@
+//! Fungible tokens as described in [by the spec](https://nomicon.io/Standards/Tokens/FungibleToken).
```

Comment:
this change somehow results in this section
![image](https://github.com/user-attachments/assets/7f5e9f2f-4804-471a-8c36-14d33af13a00)
rendered correctly vs
![image](https://github.com/user-attachments/assets/659a8841-3cc2-4bdb-beac-2da819bde531)
at https://docs.rs/near-contract-standards/5.6.0/near_contract_standards/fungible_token/index.html